packages/rancher-sriov/charts/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: sriov
+description: Install SR-IOV operator, cni and device plugin.
+version: 1.0.0
+appVersion: 1.0.0
+home: https://github.com/k8snetworkplumbingwg/sriov-network-operator
+keywords:
+  - sr-iov
+  - Networking
+sources:
+  - https://github.com/rancher/charts
+maintainers:
+  - name: Rancher Labs
+    email: charts@rancher.com
+annotations:
+  catalog.cattle.io/certified: rancher
```

Comment:
Just to confirm, does this need a custom UI? e.g. `catalog.cattle.io/ui-component`

Reply:
I don't think so